src/io/flutter/performance/FlutterPerformanceView.java

```diff
@@ -212,9 +224,15 @@ private void addPerformanceViewContent(FlutterApp app, ToolWindow toolWindow) {
     if (debugConnectionAvailable) {
       state.disposable = Disposer.newDisposable();
 
-      addFPSTab(runnerTabs, app, toolWindow, true);
-      addMemoryTab(runnerTabs, app, state);
-      addWidgetRebuildsTab(runnerTabs, app, state);
+      // Create the three FPS, memory, and widget recount areas.
+      final PerfFPSTab perfTab = new PerfFPSTab(app, this);
+      perfViewsPanel.add(perfTab);
+
+      final PerfMemoryTab memoryTab = new PerfMemoryTab(app, this);
+      perfViewsPanel.add(memoryTab);
+
+      final PerfWidgetRebuildsTab rebuildsTab = new PerfWidgetRebuildsTab(app, this);
+      perfViewsPanel.add(rebuildsTab);
```

Comment:
comment about class naming as "tab" still stands. Since these are no longer tabs, maybe they should be replaces with `PerfFPSSection`, `PerfMemorySection`, `PerfWidgetRebuildsSection`

Reply:
Ah, yeah - I'll rename away from 'tab'.